modules/nw-nad-cr.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/creating-primary-nad.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-nad-cr_{context}"]
+== Configuration for an additional network attachment
+
+An additional network is configured by using the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group.
+[IMPORTANT]
+====
+Do not store any sensitive information or a secret in the `NetworkAttachmentDefinition` object because this information is accessible by the project administration user.
```

Comment:
I actually don't understand this callout because you have to be an Administrator to create a NAD. Checking with Engr @tssurya wdyt git rid of this callout?

Reply:
yeah I don't understand the sentence either, let's get rid of it..